src/WebJobs.Script.WebHost/Security/Authorization/NamedAuthLevelAuthorizationHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using static Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization.AuthUtility;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization
+{
+    public class NamedAuthLevelAuthorizationHandler : AuthorizationHandler<AuthLevelRequirement, string>
```

Comment:
Add tests for this class. I see you added tests for your other handler (FunctionAuthorizationHandler)